pkg/controller/v1alpha2/experiment/util/status_util.go

```diff
@@ -0,0 +1,154 @@
+/*
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
+
+	experimentsv1alpha2 "github.com/kubeflow/katib/pkg/api/operators/apis/experiment/v1alpha2"
+	trialsv1alpha2 "github.com/kubeflow/katib/pkg/api/operators/apis/trial/v1alpha2"
+)
+
+var log = logf.Log.WithName("experiment-status-util")
+
+const (
+	ExperimentCreatedReason   = "ExperimentCreated"
+	ExperimentRunningReason   = "ExperimentRunning"
+	ExperimentSucceededReason = "ExperimentSucceeded"
+	ExperimentFailedReason    = "ExperimentFailed"
+	ExperimentKilledReason    = "ExperimentKilled"
```

Comment:
Will we mark Experiment Killed somewhere? 
I can't find implementation of it here.

Reply:
Not yet implemented yet. Currently our trials do not yet support status 'Killed'